core/lib/libmbedtls/src/tee_lmd_provider.c

```diff
@@ -583,53 +683,290 @@ crypto_acipher_ecc_shared_secret(struct ecc_keypair *private_key __unused,
  ******************************************************************************/
 
 #if defined(_CFG_CRYPTO_WITH_CIPHER)
-TEE_Result crypto_cipher_alloc_ctx(void **ctx __unused, uint32_t algo __unused)
+
+static TEE_Result cipher_get_ctx_size(uint32_t algo, size_t *size)
 {
-	return TEE_ERROR_NOT_IMPLEMENTED;
+	switch (algo) {
+#if defined(CFG_CRYPTO_ECB)
+	case TEE_ALG_AES_ECB_NOPAD:
+		*size = sizeof(mbedtls_cipher_context_t);
```

Comment:
Since we're always returning the same size provided that the algo is supported this could be written more like `hash_get_ctx_size()` in `core/lib/libtomcrypt/src/tee_ltc_provider.c` instead.

Reply:
Yes, that will be more simple. But these may be one issue when all configures which relative with `mbedtls_cipher_context_t` are not enable.